modules/ROOT/pages/afm-modify-monitor.adoc

```diff
@@ -31,7 +31,7 @@ To modify a monitor from its detail page:
 . Click *More* (image:afm-ui-more-button.png[width=0.75%,height=0.75%]) next to the monitor name.
 . Click *Edit Monitor*.
 
-You can edit the endpoints' *Validate SSL Certificate* toggle setting, methods, URLs, optional headers, and assertions. See xref:create-monitor.adoc[Creating Monitors] for more detail on the fields.
+You can edit the endpoints' *Validate SSL Certificate* toggle setting, methods, URLs, optional headers, and assertions. See xref:afm-create-monitor.adoc[Creating Monitors] for more detail on the fields.
```

Comment:
```suggestion
You can edit the endpoint's *Validate SSL Certificate* toggle setting, methods, URLs, optional headers, and assertions. See xref:afm-create-monitor.adoc[Creating Monitors] for more detail on the fields.
```

Reply:
It's correct as it is. There can be multiple endpoints.